common/pom.xml

```diff
@@ -90,6 +90,29 @@
                 </executions>
             </plugin>
 
+            <plugin>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>3.0.2</version>
+                <executions>
+                    <execution>
+                        <id>copy-resources</id>
+                        <phase>package</phase>
```

Comment:
@adaykin Take a look at the CI output: this does not seem to have worked, likely due to the wrong `phase` value?
![image](https://user-images.githubusercontent.com/11096575/75987082-4a80b400-5ebd-11ea-88ed-69eeb12f45c7.png)